src/FSLibrary/StellarCoreCfg.fs

```diff
@@ -162,6 +162,7 @@ type StellarCoreCfg =
       maxSlotsToRemember: int
       maxBatchWriteCount: int
       inMemoryMode: bool
+      addArtificialDelay: int option
```

Comment:
Can you please add a comment to this field indicating that what unit of duration it expects (milliseconds, I think?)

Reply:
Done!